src/petab_gui/views/task_bar.py

```diff
@@ -73,6 +73,9 @@ def __init__(self, parent, actions):
         self.menu.addAction(actions["copy"])
         self.menu.addAction(actions["paste"])
         self.menu.addSeparator()
+        # Undo, Redo
+        self.menu.addAction(actions["undo"])
+        self.menu.addAction(actions["redo"])
```

Comment:
If possible, it would be great to change the order, the default order in many of my programs for the edit menu is: 

- undo , redo ; sep; 
- cut, copy, paste; sep
- find, replace; sep

it would be great if we could make this happen too: 

![example](https://p194.p3.n0.cdn.zight.com/items/2NueyveK/cf8c275c-7f02-4e2b-90a4-d0de9b7fe37b.jpeg?v=1f9aaed8df01c2c0ab36576cfefa1bcb)

additionally, there is an event on the undo stack, that tells you whether undo / redo is available. And it would be great if we could use them to enable / disable the action: 

<https://doc.qt.io/qt-6/qundostack.html#canRedoChanged>

Reply:
Changed Order :+1: